Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalCell.cc

```diff
@@ -80,10 +83,18 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
   double dy1 = r;
   double dy2 = 0.5 * dy1;
   double dy3 = 1.5 * dy1;
+  double dx5 = (addWaferSeparation == 0) ? 0.0 : waferSeparation * 0.5;
```

Comment:
Just wonder why don't you arrange dx1....dx7, then dy1 .... dy6.
But this is just to make in order but not necessary.

Reply:
Done